src/dashboard/Settings/components/SettingsLayout/index.js

```diff
@@ -0,0 +1,119 @@
+import classnames from 'classnames';
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Button, Col, Input, Row, Layout, Checkbox } from 'antd';
+import PropTypes from 'prop-types';
+import SidebarSettings from '../SidebarSettings';
+import ColHeader from '../../../../common/components/ColHeader';
+import IncidentType from '../SystemSettings/Components/IncidentTypeList';
+import IncidentDetails from '../SystemSettings/Components/IncidentDetails';
+import AddNewIncidentType from '../SystemSettings/Components/IncidentTypeForm/AddNewIncidentType';
+import EditIncidentTypeForm from '../SystemSettings/Components/IncidentTypeForm/EditIncidentTypeForm';
+
+/* load styles */
+import styles from './styles.css';
+
+/* local constants */
+const { Content } = Layout;
+const { Search } = Input;
+const cx = classnames.bind(styles);
```

Comment:
I suggest using `classnames` when we need to implement conditional class loading.

Reply:
Instead just use 
```suggestion
import './styles.css';

// component
<Component className="classname" />
```